package.json

```diff
@@ -4,20 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "npm-run-all -p build:ui build:semantic-ui",
     "build:changelog": "auto-changelog -p --tag-pattern '^((?!rc|test).)*$' --template changelog-template.hbs && git add CHANGELOG.md",
     "build:common:js": "cd ./packages/common && yarn build:js",
     "build:common:schema:api": "cd ./packages/common && yarn build:schema:api",
     "build:common:schema:models": "cd ./packages/common && yarn build:schema:models",
     "build:data:zip": "node ./packages/scripts/src/js/packageJsonDataFiles.js -s local",
     "build:index:specimen": "yarn start:dbs && cd ./packages/migrations && yarn elasticsearch:development:rebuild-index:specimen-search",
     "build:semantic-ui": "cd ./packages/dina-style && yarn build:docs",
+    "build:trees": "yarn build:trees:ui && yarn build:trees:common && yarn build:trees:backend",
```

Comment:
these could be parallelized like `yarn build:trees:ui & yarn build:trees:common & yarn build:trees:backend`